policy/policy-authority-data.go

```diff
@@ -40,20 +43,27 @@ type RuleSet struct {
 	Whitelist []WhitelistRule
 }
 
+type gorpDbMap interface {
```

Comment:
Nit: Using gorp in the interface name here seems like mainly a nod to history.  I would go with just `dbMap`.